src/app/loaders/express/index.ts

```diff
@@ -141,6 +143,21 @@ const loadExpressApp = async (connection: Connection) => {
   // Log intranet usage
   app.use(IntranetMiddleware.logIntranetUsage)
 
+  app.use(
+    connectDatadog({
+      method: true, // i.e. normalized path, to keep cardinality in check
+      response_code: true,
+      path: false, // !! Important: do not turn this true or the tag cardinality will explode
+      dogstatsd: new StatsD({
+        useDefaultRoute: true,
+        errorHandler: (error) => {
+          // eslint-disable-next-line no-console
+          console.log(error)
```

Comment:
under what circumstances will we encounter errors here? should we be logging a specific string here so we can set alerts for it?

Reply:
Actually, I'm not sure and I was even thinking of removing this error handler entirely 🤔 . I added it yesterday because I had trouble getting data, and I was suspecting some app-level issue, but I had just misconfigured the agent (shame on me, I didn't read Alwyn's sample properly! 😑 )

Now that you mention it I think I will remove it. being unable to send metrics would likely be reflected in the metrics themselves (no data or less data?), but logging every metric push error would just pollute logs.

Reply:
removed